src/core/tests/type_prop/string_tensor_pack.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/string_tensor_pack.hpp"
+
+#include <gtest/gtest.h>
+
+#include "common_test_utils/test_assertions.hpp"
+#include "common_test_utils/type_prop.hpp"
+
+namespace ov {
+namespace test {
+
+using ov::op::v0::Parameter;
+using testing::HasSubstr;
+
+class TypePropStringTensorPackTestSuite : public ::testing::TestWithParam<ov::PartialShape> {};
+
+TEST_P(TypePropStringTensorPackTestSuite, TypePropStringTensorPackTestSuite) {
+    const auto& indices_shape = GetParam();
+    const auto begins = std::make_shared<Parameter>(element::i32, indices_shape);
+    const auto ends = std::make_shared<Parameter>(element::i32, indices_shape);
+    const auto symbols = std::make_shared<Parameter>(element::u8, PartialShape{100});
```

Comment:
Why shape is fixed and set to `100`. When e.g two inputs are vectors of 2 elements these shape should also have be vector of 2 elements.

Reply:
I changed the `symbols` shape to `PartialShape{Dimension::dynamic()}`, as in this test it does not factor into the asserts.